lib/ansible/plugins/connection/ssh.py

```diff
@@ -167,7 +171,7 @@
               version_added: '2.7'
           cli:
             - name: ssh_extra_args
-      retries:
+      reconnection_retries:
```

Comment:
Was this rename purposeful in this backport?  It won't work without changing the `get_option` call.